source/_patterns/01-elements/_form-elements.scss

```diff
@@ -33,6 +33,38 @@
 	}
 }
 
+// Validation
+// We need to differentiate here for select elements, that we couldn't handle by :placeholder-shown
+// TODO: maybe we could totally restructure this, work with a :not(select) instead, differentiate in between user-invalid and provide a fallback (https://github.com/db-ui/mono/pull/893), etc.
```

Comment:
maybe we could totally restructure this, work with a :not(select) instead, differentiate in between user-invalid and provide a fallback (https://github.com/db-ui/mono/pull/893), etc.